relay-event-schema/src/processor/impls.rs

```diff
@@ -254,3 +258,50 @@ process_tuple!(T1, T2, T3, T4, T5, T6, T7, T8, T9);
 process_tuple!(T1, T2, T3, T4, T5, T6, T7, T8, T9, T10);
 process_tuple!(T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11);
 process_tuple!(T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12);
+
+impl ProcessValue for Value {
```

Comment:
This was previously derived and is now implemented manually. The manual implementation is slightly more performant as it avoids cloning the processing state when traversing into the branches of the `Value` type.